lms/static/js/edxnotes/views/notes_page.js

```diff
@@ -31,7 +31,8 @@ define([
                 el: this.el,
                 collection: this.collection,
                 tabsCollection: this.tabsCollection,
-                scrollToTag: scrollToTag
+                scrollToTag: scrollToTag,
+                createHeaderFooter: true
```

Comment:
nope. we only need to pass the `perPage` to `searchResultsView` and from there it will be passed to `SearchBoxView` to initialize the notes collection.